cityehr-quick-start-guide/src/main/xslt/create-topic-html.xslt

```diff
@@ -25,15 +34,29 @@
         </xsl:call-template>
       </head>
       <body>
-        <div id="top-nav">
+        <nav id="top-nav">
           <xsl:apply-templates select="." mode="top-nav"/>
-        </div>
+        </nav>
+
+        <!-- Petal Edit Button -->
+        <section>
+          <xsl:variable name="petal-source-file-uri" select="com:document-uri(.)" />
+          <xsl:variable name="tokens" select="tokenize($petal-source-file-uri, '/')"/>
+          <xsl:variable name="orig-file-name" select="$tokens[last()]"/>
+          <xsl:variable name="petal-source-file" select="substring-after($petal-source-file-uri, $petal-github-repo-name || '/')" />
+          <xsl:variable name="petal-file-name" select="hcom:dita-filename-to-html($orig-file-name)" />
+          <xsl:variable name="petal-full-url" select="concat($petal-base-url, '?ghrepo=', $petal-github-base-url, '/', $petal-github-org-name, '/', $petal-github-repo-name, '&amp;source=', $petal-source-file, '&amp;branch=', $petal-github-branch, '&amp;referer=', $petal-referrer-url, $petal-file-name)" />
+          <a href="{ $petal-full-url }">
+            <input type="button" value="Edit this page" />
+          </a>
+        </section>
+
         <article>
           <xsl:apply-templates select="element()" mode="body"/>
         </article>
-        <div id="bottom-nav">
+        <nav id="bottom-nav">
```

Comment:
Why is this PR changing something unrelated to its purpose?

Reply:
See above.